src/manager/command_parser/sc_component_manager_command_parser.hpp

```diff
@@ -19,11 +19,44 @@ class ScComponentManagerParser
   static std::pair<std::string, CommandParameters> Parse(std::string const & command)
   {
     std::string const COMPONENTS_COMMAND_PREFIX = "components";
+    std::vector<std::string> const COMMAND_COMPONENTS_INIT = {"components init", "ci", "comp init"};
+    std::vector<std::string> const COMMAND_COMPONENTS_SEARCH = {"components search", "cs", "comp search", "comp s"};
+    std::vector<std::string> const COMMAND_COMPONENTS_INSTALL = {"components install", "cinst","comp inst"};
+    std::vector<std::vector <std::string>> const COMMAND_LIST = {COMMAND_COMPONENTS_INIT, COMMAND_COMPONENTS_SEARCH, COMMAND_COMPONENTS_INSTALL};
     size_t const COMMAND_KEYWORDS_SIZE = 2;
 
+    std::string cutCommand;
+    size_t firstPapameter = command.find('-');
+    if (firstPapameter != std::string::npos)
+    {
+      cutCommand = command.substr(0, firstPapameter - 1);
+    }
+    else 
+    {
+      cutCommand = command;
+    }
+    size_t endOfCommandPos = cutCommand.find_last_not_of(' ');
+    if (endOfCommandPos != std::string::npos) 
+    {
+        cutCommand.erase(endOfCommandPos + 1);
+    }
+
+    std::string fullCommand = command;
+    for (int indexCommand = 0; indexCommand < COMMAND_LIST.size(); indexCommand++)
+    {
+      for (int indexReducedCommand = 1; indexReducedCommand < COMMAND_LIST[indexCommand].size(); indexReducedCommand++)
+      {
+        if (cutCommand == COMMAND_LIST[indexCommand][indexReducedCommand])
+        {
+          fullCommand.replace(0, cutCommand.size(), COMMAND_LIST[indexCommand][0]);
+          break;
```

Comment:
В этом случае мы выйдем из второго for и будем продолжать искать возможную замену. Не страшно, но чем больше список, тем больше итераций. Наверное, можно добавить такое или закончить поиск иначе
```suggestion
          indexCommand = COMMAND_LIST.size();
          break;
```
Еще Максим просил проверить работают ли сокращения. Проверила, все круто